### PR TITLE
powerline-go: Add version 1.21.0

### PR DESCRIPTION
**Pull request by i-ghost**

### CODE REVIEW

**Review by CryoRenegade — CHANGES_REQUESTED** [review]
Add a newline at the end of the file.

**Comment by pratikpc** [comment]
@i-ghost can you add check-ver to support auto-updation?

**Comment by i-ghost** [comment]
@pratikpc That's done now, sorry for the delay.

**Comment by i-ghost** [comment]
@rashil2000 Changes implemented, rebased on latest upstream `master`, and squashed commits.

File: bucket/powerline-go.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/justjanne/powerline-go/releases/download/v1.21.0/powerline-go-windows-amd64#/powerline-go.exe",
+            "hash": "F5B495807C9F2BAEFE1D9D2F584AD3D1AFDFD4EDA96C4828B1B458D9908BCD8C"
+        }
+    },
+    "bin": "powerline-go.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/justjanne/powerline-go/releases/download/v$version/powerline-go-windows-amd64#/powerline-go.exe"
```
Comment (pratikpc) [inline bucket/powerline-go.json:15]: Needs architecture and 64bit

**Review by pratikpc — CHANGES_REQUESTED** [review]

**Comment by pratikpc** [comment]
Are the GitHub Actions broken?

**Review by pratikpc — APPROVED** [review]

**Pull request merged**